frontend/packages/core/src/card.tsx

```diff
@@ -51,23 +49,65 @@ const StyledCardHeaderContainer = styled.div({
   background: "#EBEDFB",
 });
 
-interface CardHeaderProps extends Pick<MuiCardHeaderProps, "avatar" | "title"> {
+const StyledCardHeader = styled(Grid)({
+  padding: "16px",
+  minHeight: "72px",
+  margin: "0px",
+  width: "100%",
+});
+
+const StyledCardHeaderAvater = styled.div({
+  marginRight: "16px",
+  fontSize: "24px",
+});
+
+const StyledDivider = styled(Divider)({
```

Comment:
Should we sync with Kanchan on if we want this as a component?

Reply:
yeah, left a comment for Kanchan in figma